internal/powerplatform/services/data_record/resource_data_record.go

```diff
@@ -51,6 +50,7 @@ func (r *DataRecordResource) Schema(ctx context.Context, req resource.SchemaRequ
 		Description:         "PowerPlatform Data Record Resource",
 		MarkdownDescription: "Resource for managing PowerPlatform Data Record",
 		Attributes: map[string]schema.Attribute{
+			//todo would be good to add the record url here as well or set is as "id"
```

Comment:
What is the record url? docs?

Reply:
record url so you have something like http://CRM/data/api/contacts(1111-2222-.....3333)

Reply:
yes that would be good, let's add that instead of a comment